### ISSUE TITLE
'Add-Link-Overlay' search result list - navigate with Up & Down Arrow Keys crashes block preview

### ISSUE THREAD

**Issue by s-light** [WordPress/gutenberg#10858]
**Describe the bug**
Add-Link-Overlay: Search for a other blog entry to link to and then try to select one of the results from the list with the key 'Arrow-Down' crashes the blog preview.

**To Reproduce**
*! only use the keyboard !*
1. create a paragraph block
2. add some text
3. navigate to the word you want to link and select it (Ctrl+Arrow-Rigth) (screenshot 1)
4. use Ctrl+K to open 'Add Link' overlay
5. start typing your search text
6. list of results shows up (screenshot 2)
7. use 'Arrow-Down' key to select (first) result from list
8. block preview crashes. (screenshot 3)

**Expected behavior**
i expect to be able to select the entry from the search-result list with the up and down arrow keys, and than confirm the selection with 'Enter'

**Screenshots**

1. starting with selected word to be linked (Ctrl + Shift + ArrowRight)
 ![gutenberg__selected](https://user-images.githubusercontent.com/1340319/47266921-106b7800-d53d-11e8-8f59-aa5d8300ffe2.png)
1. open Link Dialog with Ctrl+K and entered search string
 ![gutenberg__searchresults](https://user-images.githubusercontent.com/1340319/47266922-14979580-d53d-11e8-8b9d-ea90f09c556f.png)
1. now using Key 'Arrow-Down' to select first find in list -yields this crash:
 ![gutenberg__blockpreviewcrash](https://user-images.githubusercontent.com/1340319/47266924-18c3b300-d53d-11e8-9142-b13780a86309.png)

**Desktop:**
 - OS: Kubuntu 18.04
 - Browser Firefox
 - Version 62.0.3 (64bit)

**Additional context**
- Please add the version of Gutenberg you are using in the description.
error in console:
![image](https://user-images.githubusercontent.com/1340319/47266828-144aca80-d53c-11e8-9f09-89c8720397b3.png)
```
14:14:07.760 TypeError: "o is undefined"
t http://xy.eu/wp-content/plugins/gutenberg/build/editor/index.js?ver=1540114991:12:35439
s http://xy.eu/wp-content/plugins/gutenberg/build/editor/index.js?ver=1540114991:12:35385
offset http://xy.eu/wp-content/plugins/gutenberg/build/editor/index.js?ver=1540114991:12:38487
exports http://xy.eu/wp-content/plugins/gutenberg/build/editor/index.js?ver=1540114991:12:39826
value http://xy.eu/wp-content/plugins/gutenberg/build/editor/index.js?ver=1540114991:50:143614
jc http://xy.eu/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:134:236
gc http://xy.eu/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:127:223
vb http://xy.eu/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:126:230
ub http://xy.eu/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:126:65
wg http://xy.eu/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:137:182
Ze http://xy.eu/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:41:24 
react-dom.min.82e21c65.js:110:57 
```

**Comment by annestk** [comment]
Duplicate of #10689 - there'll be a fix (#10716) included in the next update 😊

**Comment by s-light** [comment]
ok - sorry for the duplicate. did not find the previous bug (i think i had limited my search to open bugs..)
